.gitignore

```diff
@@ -32,5 +32,4 @@ bin
 **/*.key
 **/*.pem
 **/*.csr
-**/server-csr.json
```

Comment:
I don't get the point of removing this.

Reply:
The point was that my server-csr.json in the nats folder was being ignored but it's necessary to generate the certificates. I re-added it, but making sure that the required files are not ignored.